### PR TITLE
pr 5

### PR DESCRIPTION
**Pull request by techlewi**

### CODE REVIEW

**Comment by callumok2004** [comment]
Make sure to provide screenshots to show the presence works!

File: GamingAnthems/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    "en": "No1 Gaming Radio Station"
+  },
+  "url": [
+    "gaminganthems.com",
+    "raidstop.co.uk"
+    "raidstop.com"
+  ],
+  "version": "1.0.0",
+  "logo": "https://cdn.discordapp.com/attachments/681603738468614247/692449497921355846/logo_blacvk.png",
+  "thumbnail": "https://cdn.discordapp.com/attachments/681603738468614247/692449501197369404/logo_pink_and_black.png",
```
Comment (Bas950) [inline GamingAnthems/dist/metadata.json]: upload to imgur

File: GamingAnthems/dist/metadata.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  "color": "#000000",
+  "tags": [
+    "No1",
+    "Gaming",
+    "Radio",
+    "game",
+    "Music"
+  ],
```
Comment (Bas950) [inline GamingAnthems/dist/metadata.json]: tags should be in lowercase

File: GamingAnthems/dist/Presence.js [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+    // element grabs //
+    // api calls //
+    // variable sets //
+}
+
+setInterval(10000, myOutsideHeavyLiftingFunction);
+//Run the function seperate from the UpdateData event every 10 seconds to get and set the variables which UpdateData picks up
+
+*/
+
```
Comment (Bas950) [inline GamingAnthems/dist/Presence.js]: remove all the comments

**Comment by techlewi** [comment]
> It also appears the one of the urls does not lead to a site, another one goes to a wordpress site, and the last one is the only one that leads anywhere related to gaming anthems, as it has a built in player.

yes i know it is if we move we own all the domains

File: GamingAnthems/dist/metadata.json [hunk 4]
```diff
@@ -13,15 +13,15 @@
     "raidstop.com"
   ],
   "version": "1.0.0",
-  "logo": "https://cdn.discordapp.com/attachments/681603738468614247/692449497921355846/logo_blacvk.png",
-  "thumbnail": "https://cdn.discordapp.com/attachments/681603738468614247/692449501197369404/logo_pink_and_black.png",
+  "logo": "https://imgur.com/9PEupja",
+  "thumbnail": "https://imgur.com/j7u9H9X",
```
Comment (Bas950) [inline GamingAnthems/dist/metadata.json]: you need the i.imgur links and ending with .png / .jpg

Comment (techlewi) [reply GamingAnthems/dist/metadata.json]: https://user-images.githubusercontent.com/51294309/77699256-fcd20580-6fa9-11ea-93cc-1d0584d821bc.png

Comment (techlewi) [reply GamingAnthems/dist/metadata.json]: it works

**Comment by techlewi** [comment]
https://user-images.githubusercontent.com/51294309/77699256-fcd20580-6fa9-11ea-93cc-1d0584d821bc.png

**Comment by techlewi** [comment]
> Upload proof of it working

https://user-images.githubusercontent.com/51294309/77699256-fcd20580-6fa9-11ea-93cc-1d0584d821bc.png

File: GamingAnthems/dist/metadata.json (hunk 4 above)
Comment (Bas950) [reply GamingAnthems/dist/metadata.json]: but they arent the direct image links, you need those

**Comment by techlewi** [comment]
> Upload it here, not in an external link.

Capture

it wont let me i do it but it just turns it into the link

File: GamingAnthems/dist/metadata.json [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+    "en": "No1 Gaming Radio Station"
+  },
+  "url": [
+    "gaminganthems.com",
+    "raidstop.co.uk",
+    "raidstop.com"
+  ],
+  "version": "1.0.0",
+  "logo": "https://imgur.com/9PEupja",
+  "thumbnail": "https://imgur.com/j7u9H9X",
```
Comment (Bas950) [inline GamingAnthems/dist/metadata.json]: still not the i.imgur links with ending .png / .jpg

Comment (techlewi) [reply GamingAnthems/dist/metadata.json]: it still works tho
ill change it tho

Comment (Bas950) [reply GamingAnthems/dist/metadata.json]: these links wont work on your website, they need to be the direct image links

File: GamingAnthems/dist/metadata.json [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+  },
+  "url": [
+    "gaminganthems.com",
+    "raidstop.co.uk",
+    "raidstop.com"
+  ],
+  "version": "1.0.0",
+  "logo": "https://i.imgur.com/9PEupja.png",
+  "thumbnail": "https://i.imgur.com/j7u9H9X.png",
+  "color": "#000000",
```
Comment (Alanexei) [inline GamingAnthems/dist/metadata.json]: Use a different color, the site offers other colors besides black. It adds more flare.

File: GamingAnthems/dist/metadata.json [hunk 7]
```diff
@@ -13,8 +13,8 @@
     "raidstop.com"
   ],
   "version": "1.0.0",
-  "logo": "https://imgur.com/9PEupja",
-  "thumbnail": "https://imgur.com/j7u9H9X",
+  "logo": "https://i.imgur.com/9PEupja.png",
+  "thumbnail": "https://i.imgur.com/j7u9H9X.png",
   "color": "#000000",
```
Comment (Bas950) [inline GamingAnthems/dist/metadata.json]: sorry I only just noticed it, but the thumbnail should be an image of the website, if you dont have an actual thumbnail.

Comment (techlewi) [reply GamingAnthems/dist/metadata.json]: ??

Comment (frozaum) [reply GamingAnthems/dist/metadata.json]: it can be a promotional image as well

Comment (frozaum) [inline GamingAnthems/dist/metadata.json]: not necessary, but i would recommend you to change the color

File: GamingAnthems/dist/metadata.json (hunk 6 above)
Comment (Alanexei) [reply GamingAnthems/dist/metadata.json]: Not required

**Comment by techlewi** [comment]
they are the colours we want can we just leave it as that

File: GamingAnthems/dist/metadata.json (hunk 4 above)
Comment (techlewi) [reply GamingAnthems/dist/metadata.json]: done

**Comment by techlewi** [comment]
anythin else?

**Comment by techlewi** [comment]
> Upload a better thumbnail, it should be a wide image, an actual thumbnail or a screenshot of the website

i did

**Comment by techlewi** [comment]
hey i think the deep scan bot is broke thats not my code!

**Comment by techlewi** [comment]
why on hold?

**Comment by doomlerd** [comment]
Hello @techlewi, your PR is not merged yet because we're making some changes in favor of quality assurance which will require your PR to be re-made, make sure to fix the [current reported issues](https://deepscan.io/dashboard/#view=project&tid=6792&pid=8925&bid=113944&subview=pull-request&prid=300536) and we'll inform you what to do next ( to comply with the new changes ), I'll keep this on hold now until we finish.

For reference : #1248 ( rebase #1258 )

**Comment by doomlerd** [comment]
Sorry for the wait, now that the repo is unlocked you have to recreate the PR with these changes :
format your code with either spaces or tabs and using only 2 or 4 in tab width then recreate the PR for the new checks to start.

**Pull request closed without merging**